client/templates/other/welcome.html

```diff
@@ -40,13 +40,17 @@ <h2>Getting started</h2>
                 <a href="https://codebuddiesmeet.herokuapp.com">here</a>.</li>
               <li>Don't be shy! Introduce yourself in the #introduce-yourself Slack channel, or on our forum at 
                 <a href="https://forum.codebuddies.org/t/introduce-yourself-here/38/7">https://forum.codebuddies.org/t/introduce-yourself-here/38/7</a>, and welcome fellow new members.
-                The more we know about each other, the better we can connect and collaborate. :)</li>
+                The more we know about each other, the better we can connect and collaborate. :)
+                <img src="/images/slack-introduce-yourself.jpg" class="img-responsive" id="slack-introduce-yourself">
+              </li>
               <li>You'll automatically be invited to a couple of default Slack channels. You can browse all our public Slack 
                 channels by clicking on the "Channels" link in the sidebar. Make sure to read our <a href="https://codebuddies.org/slack">CodeBuddies Slack 
                 Etiquette for General Members</a>.</li>
               <li>Log into our website at <a href="https://codebuddies.org/">http://codebuddies.org</a> using your Github or new CodeBuddies Slack account 
                 and take a look around -- or join a hangout! Hangouts are a way to study silently or pair program live with other members of the community. 
-                There are three types of hangouts you can start: silent, teaching, or collaborative.</li>
+                There are three types of hangouts you can start: silent, teaching, or collaborative.
+                <img src="/images/join-hangouts.png" width="100%" id="join-hangouts"/>
```

Comment:
Could we replace that screenshot with this new file?

![screen shot 2018-10-28 at 3 50 53 pm](https://user-images.githubusercontent.com/4512699/47623032-6013eb80-dac9-11e8-9d9e-8370b97691b9.jpg)